opm/autodiff/FlowMainEbos.hpp

```diff
@@ -700,6 +777,9 @@ namespace Opm
         Grid& grid()
         { return ebosSimulator_->gridManager().grid(); }
 
+        const Grid& globalGrid()
+        { return *globalGrid_; }
```

Comment:
didn't you once mention that creating a `CpGrid` comes down to a shallow copy? in this case an alternative that should not introduce new methods would be

```c++
globalGrid_ = grid();
globalGrid.switchToGlobalView();
```